db/data/db.json

```diff
@@ -130,32 +191,6 @@
                 "col": 7,
                 "reservation": 0
             }
-        ],
-        "room_configuration": [
```

Comment:
Room configs are required for defining the room's layout.

Reply:
Thanks for your feedback, fixed in #[526ac45 commit](https://github.com/jkutkutOrg/Android-CineHub/pull/36/commits/526ac4504f30a1aa4cb49e0a8976d81bc90af792)